Emby.Dlna/Profiles/DefaultProfile.cs

```diff
@@ -14,9 +14,9 @@ public DefaultProfile()
 
             ProtocolInfo = "http-get:*:video/mpeg:*,http-get:*:video/mp4:*,http-get:*:video/vnd.dlna.mpeg-tts:*,http-get:*:video/avi:*,http-get:*:video/x-matroska:*,http-get:*:video/x-ms-wmv:*,http-get:*:video/wtv:*,http-get:*:audio/mpeg:*,http-get:*:audio/mp3:*,http-get:*:audio/mp4:*,http-get:*:audio/x-ms-wma*,http-get:*:audio/wav:*,http-get:*:audio/L16:*,http-get:*image/jpeg:*,http-get:*image/png:*,http-get:*image/gif:*,http-get:*image/tiff:*";
 
-            Manufacturer = "Emby";
```

Comment:
Why only change the default profile, instead of all profiles? Maybe do all of them at once in a separate PR . They are located in the Xml dir.